src/t8_netcdf/t8_nc_mesh.hxx

```diff
@@ -18,4 +40,99 @@ std::pair<t8_forest_t, int>
 t8_nc_build_initial_congruent_mesh (const t8_nc_geo_domain_t& domain, const t8_nc_data_layout_t initial_layout,
                                     sc_MPI_Comm comm);
 
+/* Checking if the last bit is a zero, because then it is possible to be a multiple of 2. */
+inline bool
+t8_check_if_last_bit_is_zero(const t8_gloidx_t byte)
```

Comment:
```suggestion
t8_check_if_lsb_bit_is_zero(const t8_gloidx_t value)
```
I think we should rename the function parameter, since "byte" is not perfectly suiting.